sdk/core/azure-core-amqp/vcpkg.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "azure-core-amqp",
+  "version-string": "1.0.0-beta.1",
+  "supports": "!uwp",
+  "dependencies": [
+    {
+      "name": "azure-core-cpp",
+      "default-features": false
+    },
+    {
+      "name": "azure-uamqp-c",
+      "platform": "!uwp"
+    },
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    },
+    {
+      "name": "azure-identity-cpp"
+    },
+    {
+      "name": "gtest"
```

Comment:
The source project shouldn't depend on gtest.

Reply:
The tests depend on gtest, so the source has to depend on gtest.